boot_shutdown_interface/include/boot_shutdown_interface/boot_shutdown_interface_core.hpp

```diff
@@ -38,11 +39,13 @@ class BootShutdownInterface : public rclcpp::Node
 
 private:
   rclcpp::Publisher<EcuState>::SharedPtr pub_ecu_state_;
+  rclcpp::Subscription<std_msgs::msg::String>::SharedPtr sub_bagpacker_topic_;
```

Comment:
@v-nakayama7440-esol The boot_shutdown_tools will be used for other ECUs other than Sub ECU.
I believe this specific processing to Sub ECU should not be added.
Could you consider to realize common use of this tool?

Reply:
@ito-san 
I made all settings for subscribe topic to ROS parameters.
ded3d5f301a70ca786996b4725ae9a9a17be45ae